pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1262,6 +1263,12 @@ public void openLedgerFailed(ManagedLedgerException exception, Object ctx) {
                     topicLevelOffloadPolicies,
                     OffloadPolicies.oldPoliciesCompatible(nsLevelOffloadPolicies, policies.orElse(null)),
                     getPulsar().getConfig().getProperties());
+
+            if (offloadPolicies != null && serviceConfig.getManagedLedgerDataReadPriority() != null) {
+                offloadPolicies.setManagedLedgerOffloadedReadPriority(
```

Comment:
Maybe we should check whether offloadPolicies already have `ManagedLedgerOffloadedReadPriority` value.

Reply:
I removed these lines since the initialize progress should already be finished by ` OffloadPolicies.mergeConfiguration`.